docs/querying/raw-queries.mdx

```diff
@@ -483,11 +485,32 @@ Attributes support a shorthand syntax for casting. See [Casting syntax in `sql.a
 
 :::
 
+### `sql.uuidV4` & `sql.uuidV1`
+
+In supported dialects, using `sql.uuidV4` and `sql.uuidV1` will generate the dialect-specific function to generate a UUID.
+In unsupported dialects, using these functions, except as [the default value of an attribute](../models/data-types.mdx#built-in-default-values-for-uuid), will throw an error.
+
+```ts
+sequelize.query(sql`INSERT INTO users (id) VALUES (${sql.uuidV4()})`);
+```
+
+These functions are supported by the following dialects:
+
+```sql
+-- postgres
+INSERT INTO users (id) VALUES (gen_random_uuid())
+
+-- mssql
+INSERT INTO users (id) VALUES (NEWID())
+```
```

Comment:
Isn't this included in the table already?

Reply:
Reworked